src/protagonist/Engine/Ingest/Timebased/Transcode/ElasticTranscoder.cs

```diff
@@ -76,24 +78,29 @@ public class ElasticTranscoder : IMediaTranscoder
     {
         var asset = context.Asset;
         var assetId = context.AssetId;
-        var technicalDetails = asset.FullImageOptimisationPolicy.TechnicalDetails;
-        var outputs = new List<CreateJobOutput>(technicalDetails.Length);
+        var timeBasedPolicies = asset.ImageDeliveryChannels.Where(i => i.Channel == AssetDeliveryChannels.Timebased)
+            .Select(x => JsonSerializer.Deserialize<List<string>>(x.DeliveryChannelPolicy.PolicyData))
+            .SelectMany(i => i).ToList();
+        var outputs = new List<CreateJobOutput>();
 
-        foreach (var technicalDetail in technicalDetails)
+        foreach (var timeBasedPolicy in timeBasedPolicies)
         {
             var mediaType = context.Asset.MediaType;
-            var (destinationPath, presetName) =
-                TranscoderTemplates.ProcessPreset(mediaType, assetId, technicalDetail, jobId);
-
+            
             // TODO - handle empty path/presetname

```

Comment:
Can this TODO go now?

Reply:
For both TODO's I think I left this in as I didn't actually change what happened here - when I first looked at it I did think it was being handled though, so will remove